src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,96 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 1 << 4;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int GROW_FACTOR = 2;
+    private Node<K, V>[] table;
+    private int size;
+    private int threshold;
```

Comment:
You can don't use it, simply referencing to `DEFAULT_INITIAL_CAPACITY * DEFAULT_LOAD_FACTOR`.